src/test/spec/run_command.rs

```diff
@@ -10,6 +10,8 @@ async fn run_unified() {
         .skip_tests(&[
             // TODO re: RUST-1649: fix withTransaction for new test runner
             "attaches transaction fields to given command",
+            "always attaches $db and implicit lsid to given command and omits default readPreference",
```

Comment:
"attaches transaction fields to given command" was skipped because we don't have an implementation of `withTransaction` for the unified runner, but it doesn't look like these use that - I think these were valid failures that need investigation.